kura/org.eclipse.kura.container.orchestration.provider/src/main/java/org/eclipse/kura/container/orchestration/provider/impl/ContainerOrchestrationServiceImpl.java

```diff
@@ -705,6 +705,8 @@ private HostConfig containerPortManagementHandler(ContainerConfiguration contain
                     } catch (IllegalArgumentException e) {
                         logger.error("Invalid internet protocol: {}. Using TCP.", port.getInternetProtocol());
                     }
+                } else {
+                    logger.error("Null internet protocol: {}. Using TCP.", port.getInternetProtocol());
```

Comment:
If the protocol is null this will log `Null internet protocol: null. Using TCP`, maybe the second `null` is not necessary, we could also switch the log level to warn, since it does not seem to be an unrecoverable error.

Reply:
This comment has not been completely addressed

Reply:
sry I don't understand your suggestion, can you make a suggestion edit ? It will be faster

Reply:
Nevermind, I didn't notice that you had removed the log entry altogheter.